osu.Game/Graphics/UserInterface/OsuPasswordTextBox.cs

```diff
@@ -28,19 +28,15 @@ public partial class OsuPasswordTextBox : OsuTextBox, ISuppressKeyEventLogging
 
         protected override bool AllowUniqueCharacterSamples => false;
 
-        protected override bool AllowClipboardExport => false;
-
-        protected override bool AllowWordNavigation => false;
```

Comment:
can be done because of https://github.com/ppy/osu-framework/blob/416718c231aaeb31cd2b9c64e0388688eef2fd36/osu.Framework/Graphics/UserInterface/TextBox.cs#L55-L63